src/js/lang_redirection.js

```diff
@@ -3,89 +3,90 @@ const redirection = function() {
     let defaultLang = "fr-FR";
 
     /**
-     * @description Little verification for mozilla because navigitor.language returns just "fr" and not "fr-FR", but for you it's same things.
-     * * It's same if your navigator is in english but i'm a baguette developpeur.
+     * @description Small check for mozilla because navigitor.language returns just "fr" and not "fr-FR", but it doesn't change anything for you.
+     * * It's the same if your navigator is in english but I'm a baguette developper.
      * @returns String
      */
     function getLanguage() {
-        return navigator.languages[0].length > navigator.languages[1].length ? navigator.languages[0] : navigator.languages[1]
+        return navigator.languages[0].length > navigator.languages[1].length ? navigator.languages[0] : navigator.languages[1];
```

Comment:
; is not necessary

Reply:
Consider using a consistent styling? You should either put semicolons everywhere or nowhere.